spec/source_finder/source_file_globber_spec.rb

```diff
@@ -103,21 +146,41 @@
   describe '#ruby_files_arr' do
     before do
       expect(globber).to receive(:glob)
-        .with('{Rakefile,{*,.*}.{rb,rake,gemspec},' \
-              '{src,app,config,db,lib,test,spec,feature}/**/{*,.*}.' \
-              '{rb,rake,gemspec}}')
+        .with('{Rakefile,{*,.*}.{gemspec,rake,rb},' \
+              '{app,config,db,feature,lib,spec,src,test}/**/{*,.*}.' \
+              '{gemspec,rake,rb}}')
         .and_return(['bing/baz.rb'])
     end
     subject { source_file_globber.ruby_files_arr }
     it { is_expected.to eq(['bing/baz.rb']) }
   end
 
+  describe '#js_files_arr' do
+    before do
+      expect(globber).to(receive(:glob))
+        .with('{{*,.*}.{js},{app,src,www}/**/{*,.*}.{js}}')
+        .and_return(['bing/baz.js'])
+    end
+    subject { source_file_globber.js_files_arr }
+    it { is_expected.to eq(['bing/baz.js']) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.